src/lib/util/globalInteractions.ts

```diff
@@ -70,6 +71,22 @@ export function makeOpenCasketButton(tier: ClueTier) {
 		.setEmoji('365003978678730772');
 }
 
+export function makeOpenSeedPackButton() {
+	return new ButtonBuilder()
+		.setCustomId('OPEN_SEED_PACK')
+		.setLabel('Open Seed Pack')
+		.setStyle(ButtonStyle.Secondary)
+		.setEmoji('977410792754413668');
+}
+
+export function canRunAutoContract() {
```

Comment:
this function name doesnt make sense.

Reply:
Sorry copied from the button created in the minion status command. Have altered it now.